vue-app/src/views/VerifyLanding.vue

```diff
@@ -84,8 +92,12 @@ export default class VerifyLanding extends Vue {
   }
 
   get isRoundFullOrOver(): boolean {
+    const hasHitMaxContributors =
+      this.$store.state.currentRound.maxContributors <=
+      this.$store.state.currentRound.contributors
     return (
-      this.$store.getters.isRecipientRegistryFull ||
+      !hasHitMaxContributors ||
```

Comment:
```suggestion
      hasHitMaxContributors ||
```
This should check for `true` value, not `false`